src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,47 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
     public void getStatistic(String fromFileName, String toFileName) {
```

Comment:
The `getStatistic` method is currently handling multiple responsibilities such as reading from a file, calculating totals, and writing to a file. Consider breaking these tasks into separate methods to improve code readability and maintainability.